app/workers/deletes.php

```diff
@@ -95,6 +120,70 @@ protected function deleteUser(Document $document, $projectId)
         ], $this->getProjectDB($projectId));
     }
 
+    protected function deleteExecutionLogs(Document $document) 
+    {
+        $projectIds = $document->getAttribute('projectIds', []);
+        foreach ($projectIds as $projectId) {
+            if (!($projectDB = $this->getProjectDB($projectId))) {
+                throw new Exception('Failed to get projectDB for project '.$projectId, 500);
```

Comment:
You don't need HTTP related error codes here, because this is not the HTTP API scope. You can just skip that param.